pkg/cmd/gist/list/list.go

```diff
@@ -33,17 +37,49 @@ func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Comman
 
 	var flagPublic bool
 	var flagSecret bool
+	var flagFilter string
 
 	cmd := &cobra.Command{
-		Use:     "list",
-		Short:   "List your gists",
+		Use:   "list",
+		Short: "List your gists",
+		Long: heredoc.Docf(`
+			List gists from your user account.
+
+			You can use a regular expression to filter the description, file names,
+			or even the content of files in the gist using %[1]s--filter%[1]s.
+
+			Use %[1]s--include-content%[1]s to include content of files, noting that
+			this will be slower and increase the rate limit used. Instead of printing a table,
+			code will be printed with highlights.
+
+			For supported regular expression syntax, see https://pkg.go.dev/regexp/syntax
```

Comment:
Not sure if I included the surrounding `<` `>` earlier, but these are needed to ensure that links are linkable in the CLI manual.

```suggestion
			For supported regular expression syntax, see <https://pkg.go.dev/regexp/syntax>
```

Reply:
I commented about that above. They actually render as `<` and `>` in the usage docs to the terminal, though. I looked at a lot of links in the CLI and didn't see them either. IMO, prioritizing `--help` output to a terminal should be priority; your doc formatter can - and honestly should - be handling URLs anyway. Most do - even github.com's does. I know `markdown-lint` recommends `<>` around URLs but the original markdown spec from Daring Fireball doesn't necessitate them.

Reply:
Never mind. I just compared the results of:

```bash
grep -r '<https:' **/*.go
```

With:

```bash
grep -rP '(?<!<)https:' **/*.go
```

And stand corrected. Fixing...

Reply:
Part of the source of my confusion was that a URL at the bottom if `--help` on any command didn't wrap the URL. I found this in `pkg/cmd/root/help.go`:

```go
	helpEntries = append(helpEntries, helpEntry{"LEARN MORE", heredoc.Docf(`
		Use %[1]sgh <command> <subcommand> --help%[1]s for more information about a command.
		Read the manual at https://cli.github.com/manual
		Learn about exit codes using %[1]sgh help exit-codes%[1]s
	`, "`")})
```

I'll update this text as you recommend so it can hopefully get merged soon, but I do think it's worth considering whether terminal output should have `<>` surrounding URLs - IMO, looks bad and certainly inconsistent with any other CLI - or fix whatever is rendering markdown for the CLI docs.

Reply:
> I commented about that above. They actually render as `<` and `>` in the usage docs to the terminal, though. I looked at a lot of links in the CLI and didn't see them either. IMO, prioritizing `--help` output to a terminal should be priority; your doc formatter can - and honestly should - be handling URLs anyway. Most do - even github.com's does. I know `markdown-lint` recommends `<>` around URLs but the original markdown spec from Daring Fireball doesn't necessitate them.

I admit the `gh` help usage isn't consistent in every place URLs are emitted.  There have been multiple efforts to clean up as much as we could consistently do within various commands, however it isn't complete as you allude to.

As for whether `gh` contextually formats URLs differently depending on 1) `--help`, 2) `man`, or 3) web manual, I think that is reasonable as a next step.